extension/encoding/awslogsencodingextension/README.md

```diff
@@ -213,5 +225,62 @@ extensions:
 | `userIdentity.userId`                 | `user.id`                                                     |
 | `userIdentity.userName`               | `user.name`                                                   |
 
-
 All request parameters and response elements are included directly as nested maps in the attributes, preserving their original structure.
+
+#### ELB Access Log Fields
+
+ELB access log record fields are mapped this way in the resulting OpenTelemetry log:
```

Comment:
There are more fields in ELB access logs that are not mentioned below. Seems we're dropping them at the moment. Can you please add a list of the fields that are _not_ translated, and mention that they're TODO?

Reply:
I updated the documentation!